presto-pinot-toolkit/src/main/java/com/facebook/presto/pinot/query/PinotFilterExpressionConverter.java

```diff
@@ -90,6 +90,16 @@ private PinotExpression handleIn(
                         .collect(Collectors.joining(", "))));
     }
 
+    private PinotExpression handleIsNull(
+            SpecialFormExpression specialForm,
+            boolean isWhitelist,
+            Function<VariableReferenceExpression, Selection> context)
+    {
+        return derived(format("(%s %s)",
+                specialForm.getArguments().get(0).accept(this, context).getDefinition(),
+                isWhitelist ? "IS NULL" : "IS NOT NULL"));
```

Comment:
`String.format` is expensive and hasn't improved much until JDK17. When compiled to JDK8 bytecode concatenating with + will generate the correct `StringBuilder` usage, and compiling to more recent targets willl use invokedynamic. This may not be the most performance sensitive code but the difference is fairly stark.

Reply:
So instead of `format("(%s %s)", stringA, stringB)`, we use `stringA + stringB`?

Reply:
I can make this change, but `String.format` is used in multiple places across this particular component. I can pick also pick it up as part of seperate PR.

Reply:
If it's done this way everywhere I agree to maintain the style.